tests/unit/adapters/test_server_db.py

```diff
@@ -0,0 +1,79 @@
+from unittest import mock
```

Comment:
I understood that this was supposed to be an integration test, not a unit test. So the folder name should be `integration` rather than `unit`.

Reply:
I think you saw the wrong folder. There are 2 folders, unit and integration all of them inside the tests folder

Reply:
OK